PrebidMobile/PrebidMobileCore/src/test/java/org/prebid/mobile/core/PrebidTest.java

```diff
@@ -0,0 +1,276 @@
+package org.prebid.mobile.core;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import com.google.android.gms.ads.doubleclick.PublisherAdRequest;
+import com.mopub.mobileads.MoPubView;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.prebid.mobile.core.mockobjects.MockCustomEvent;
+import org.prebid.mobile.core.mockobjects.MockServer;
+import org.prebid.mobile.unittestutils.BaseSetup;
+import org.prebid.mobile.unittestutils.ServerResponsesBuilder;
+import org.prebid.mobile.unittestutils.TestConstants;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.prebid.mobile.core.PrebidDemandSettings.PREBID_BIDDER;
+import static org.prebid.mobile.core.PrebidDemandSettings.PREBID_CACHE_ID;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = 21, manifest = Config.NONE)
+public class PrebidTest extends BaseSetup {
+
+    @Before
+    public void setUp() {
+        super.setup();
+        Prebid.setTestServer(MockServer.class.getName());
+
+    }
+
+    @Test
+    public void testPrebidAttachForMoPub() throws Exception {
+        // set up bid response
+        ArrayList<BidResponse> bids = new ArrayList<>();
```

Comment:
Is there a benefit to having all these tests in one block instead of separating them out into smaller tests?

Reply:
To test that new keywords are overriding old ones.